### PR TITLE
[ADD][11.0] Adding module project_agile_jira

### PR DESCRIPTION
**Pull request by pnajman-modoolar**
Hello,

Module ``project_agile_jira`` extend [project_agile](https://github.com/OCA/project-agile/pull/4) and enables us to migrate projects and tasks from JIRA to Odoo. This is very raw module and it's only usage is for migration purpose and not to be used as a JIRA synchronizer, for that there are other modules.

Unfortunately, JIRA does not have fully covered API route to read workflow with all states and transitions,  as far as I know at this moment, and thus it's not full automated. In order for the module to be able to map JIRA task states with task stages, one should create workflow in Odoo with the same stage names as in JIRA, so the module could map them properly.

Kind Regards,
Petar

### CODE REVIEW

File: project_agile_jira/models/jira_worker.py [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+
+    @contextmanager
+    def session(self):
+        with api.Environment.manage():
+            new_cr = self.pool.cursor()
+            try:
+                yield new_cr
+                new_cr.commit()
+            except Exception as ex:
+                _logger.info("Failed to write to database!: %s", str(ex))
```
Comment (elicoidal) [inline project_agile_jira/models/jira_worker.py]: remove "!"

File: project_agile_jira/views/jira_config_views.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+<odoo>
+    <record id="view_project_agile_jira_config_form" model="ir.ui.view">
+        <field name="name">project.agile.jira.config.form</field>
+        <field name="model">project.agile.jira.config</field>
+        <field name="type">form</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button class="oe_stat_button" type="object" name="synchronize_projects" icon="fa-download" string="Synchronize Projects"/>
```
Comment (elicoidal) [inline project_agile_jira/views/jira_config_views.xml]: Can you format a bit the xml for readability?

File: project_agile_jira/README.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline project_agile_jira/README.rst:38]: Please see: https://github.com/OCA/web/pull/881#pullrequestreview-99094143

**Review by elicoidal — CHANGES_REQUESTED** [review]
thanks for the module. some format details and no functional test.
Would miss UT and DD

File: project_agile_jira/models/jira_worker.py (hunk 1 above)
Comment (pnajman-modoolar) [reply project_agile_jira/models/jira_worker.py]: Fixed

File: project_agile_jira/views/jira_config_views.xml (hunk 2 above)
Comment (pnajman-modoolar) [reply project_agile_jira/views/jira_config_views.xml]: Fixed. I hope it's ok now ...

**Pull request closed without merging**